composer.json

```diff
@@ -21,6 +20,7 @@
         "ext-json": "*"
     },
     "require-dev": {
+        "fakerphp/faker": "^1.9",
```

Comment:
I noticed p2p is using faker for something, can you double check that it's not relying on core in production for any reason?

https://github.com/impress-org/give-peer-to-peer/blob/cb9383834c234c597544e40ac3af83b1e2ee9780/src/P2P/Factories/TeamFactory.php#L5

Reply:
Good catch! I'll check.